tests/test_run_config_generator.py

```diff
@@ -50,7 +50,9 @@ def setUp(self):
     def test_generate_model_config_combinations(self):
         args = [
             'model-analyzer', 'profile', '--model-repository', 'cli_repository',
-            '-f', 'path-to-config-file', '--profile-models', 'vgg11'
+            '-f', 'path-to-config-file', '--profile-models', 'vgg11',
+            '--run-config-profile-models-concurrently-enable',
```

Comment:
Why is this test changed? It doesn't look like any behavior is changed?

Reply:
I added this here to check that the CLI is being processed correctly. There's no behavior change to the test, but in the future if someone changes this CLI option it will cause the test to fail w/ a CLI parsing error.

Reply:
I understand the desire to protect it, but I really don't like it here. It is very confusing to anyone looking at this test, trying to understand what effect these options have on it.  The testing should really go into a CLI testing area, which it appears we don't currently have. I have created a ticket for that. For now, we should revert this file.  Later, once the feature is hooked up and does something it will be protected by new tests.

Reply:
Understood and reverted.